tests/verifications/openai/fixtures/test_cases/chat_completion.yaml

```diff
@@ -0,0 +1,162 @@
+test_chat_basic:
+  test_name: test_chat_basic
+  test_params:
+    input_output:
+    - input:
+        messages:
+        - content: Which planet do humans live on?
+          role: user
+      output: Earth
+    - input:
+        messages:
+        - content: Which planet has rings around it with a name starting with letter
+            S?
+          role: user
+      output: Saturn
+    model:
+    - Llama-3.3-8B-Instruct
```

Comment:
later: will be good to keep the tests independent of the models. that way, when new providers are added, we dont have to change test specific files. we can let providers determine which tests are relevant to which of their models. 

in the far distant future: we can have model capability enums and tests that match them. that way, there wont be a need to explicitly list models anywhere.